core/src/main/scala/org/apache/spark/internal/config/Status.scala

```diff
@@ -19,8 +19,30 @@ package org.apache.spark.internal.config
 
 import java.util.concurrent.TimeUnit
 
+import org.apache.spark.network.util.ByteUnit
+
 private[spark] object Status {
 
+  val IMS_CHECKPOINT_ENABLED =
+    ConfigBuilder("spark.appStateStore.ims.checkpoint.enabled")
```

Comment:
Is `ims` short for `InMemoryStore`? How about just `spark.appStateStore.checkpoint.enabled`